Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            //Get input from the user.
```

Comment:
Contents of demo should call another function- ProcessPersonBmi, PromptBmi or similar.